Dockerfile

```diff
@@ -96,7 +94,10 @@ RUN rm -rf node_modules
 FROM base-fpm AS build-fpm
 
 WORKDIR /var/www/html
-COPY --from=build-fpm-composer --chown=www-data:www-data /var/www/html  .
+COPY --chown=www-data:www-data ./config ./config
```

Comment:
Methodology: 4. Since we're not doing `COPY . .` , selectively copy folders here.